README.md

```diff
@@ -963,6 +964,24 @@ services:
   - name: starttls-smtp-example
     url: "starttls://smtp.gmail.com:587"
     interval: 30m
+    client:
+        timeout: 5s
+    conditions:
+      - "[CONNECTED] == true"
+      - "[CERTIFICATE_EXPIRATION] > 48h"
+```
+
+
+### Monitoring a service using SSL
+If you have an email server that you want to ensure there are no problems with, monitoring it through SSL
+will serve as a good initial indicator:
```

Comment:
Any chance you could update the description? It's pretty much the same as the description for `STARTTLS`.

Also, is it really accurate to make the protocol `ssl://` when it's in fact `tls`?

Reply:
> Also, is it really accurate to make the protocol ssl:// when it's in fact tls?

You're right, `tls://` would be more accurate. I've updated it.